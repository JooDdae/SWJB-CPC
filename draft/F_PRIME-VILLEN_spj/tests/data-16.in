56533323372776921533527916187418663109732294787939789298039691724771939209987173333727175996971291405179192837456732295171712192414027493770279491564175913823123145837757187356396703907304126478123165744216469747711038273700129692549771735711935115107652992272557559197697741539345072087517936742439475189797153152323411277367779011277371920998377814719123095281427121112053202765811975711
