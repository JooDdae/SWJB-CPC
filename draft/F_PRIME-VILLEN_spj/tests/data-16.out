210
3 119 2 2 1 2 2 4 1 4 136 1 0 3 2 2 3 1 22 0 18 1090 1078 29 4 2 1 10 643 1162 24 10 1011 125 4 1 15 4 8 2 1142 1218 4 2 2 2 2 4 1 4 0 4 17 1197 0 10 0 559 22 8 1 23 4 619 2 1 10 3 0 4 0 4 0 1 8 1 13 557 660 904 1176 3 116 4 17 0 2 143 0 1 2 0 620 4 4 3 918 2 740 865 155 436 1 118 1022 260 601 839 4 15 4 0 27 144 2 901 428 373 4 4 0 4 2 3 4 0 8 2 3 0 0 3 28 844 1144 1 3 3 4 3 17 8 136 4 4 13 711 611 315 3 0 22 2 19 582 161 3 0 24 22 4 0 3 2 0 3 1 2 1 2 13 0 1 4 4 2 19 4 998 0 1 4 4 2 4 8 317 23 4 4 1023 8 0 343 3 60 586 1 0 0 0 310 307 852 8 4 3 4 0 0 